vars/updatecli.groovy

```diff
@@ -1,3 +1,4 @@
+import java.util.*
```

Comment:
I don't think this is what you meant to import

Reply:
Indeed ^^